saleor/graphql/discount/dataloaders.py

```diff
@@ -150,6 +151,71 @@ def batch_load(self, keys):
         ]
 
 
+class VoucherInfoByVoucherCodeLoader(DataLoader):
+    context_key = "voucher_info_by_voucher_code"
+
+    def batch_load(self, keys):
+        vouchers_map = (
+            Voucher.objects.using(self.database_connection_name)
+            # FIXME dataloader should not operate on prefetched data. The channel
```

Comment:
If not yet done, we should open a task for SRE/core to fix it.

Reply:
The task is already created